metricbeat/module/kubernetes/state_node/_meta/test/ksm.v1.8.0.expected

```diff
@@ -29,6 +29,9 @@
 				}
 			},
 			"status": {
+				"diskpressure": "false",
+				"memorypressure": "false",
+				"pidpressure": "false",
 				"ready": "true",
```

Comment:
For safety we could try to add an operation to convert these values to actual booleans, so we detect earlier if they are not valid values for these fields. This can be done as a follow up.

Reply:
👍🏼 would this be a breaking change though? Not sure if `status.ready` is already used in Dashboards as string 🤔 .

Reply:
Ah, yes, this can be also `unknown` :) https://github.com/elastic/beats/pull/23905/files#diff-90d2faea838af7594ed44c780fc055a9c3272091368372e81957f169b2f28bf7R13
Nothing to change then.